lib/Service/ScanService.php

```diff
@@ -31,16 +32,11 @@ public function withLogger(LoggerInterface $logger): ScanService {
 
 	/**
 	 * @return int how many files where actually processed
-	 * @throws \OCP\DB\Exception if the database platform is not supported
+	 * @throws Exception if the database platform is not supported
 	 */
 	public function run(): int {
 		$unscannedTagIsDisabled = $this->appConfig->getValueBool(Application::APP_ID, 'disableUnscannedTag');
-		$quantity = $this->appConfig->getValueInt(Application::APP_ID, 'scanQueueLength');
-		try {
-			$quantity = intval($quantity);
-		} catch (\Exception) {
-			$quantity = 5;
-		}
+		$quantity = $this->appConfig->getValueInt(Application::APP_ID, 'scanQueueLength', 50);
```

Comment:
This function can not throw and has a default value now. So the former stuff was unnecessary.

Reply:
so you sneaked in some changes here? :smile: